@vates/generator-toolbox/.USAGE.md

```diff
@@ -32,12 +32,12 @@ const wrappedGenerator = new Throttle(sourceGenerator, speed)
 
 Fork a generator. The rules ares:
```

Comment:
I spent a nontrivial amount of time on this block... I'm still not certain I understand all of the edges...